draft-ietf-rats-corim.md

```diff
@@ -1514,6 +1514,58 @@ CoRIM-based data structures define an external representation of Conceptual Mess
 Appraisal processing describes both mapping transformations and Verifier reconciliation {{sec-verifier-rec}}.
 Non-CoRIM-based data structures require mapping transformation, but these are out of scope for this document.
 
+If a CoRIM profile is specified, there are a few well-defined points in the procedure where Verifier behaviour
+depends on the profile.
+The CoRIM profile MUST provide a description of the expected Verifier behavior for each of those well-defined points.
+
+Verifier implementations MUST exhibit the same externally visible behavior as described in this specification.
+They are not required to use the same internal representation described by this specification.
+
+## Appraisal Procedure {#sec-appraisal-procedure}
+
+The appraisal procedure is divided into several logical phases.
+
+### **Phase 1**: Input Validation and Transformation
+
+During Phase 1, Conceptual Message inputs are cryptographically validated, such as checking digital signatures.
+Inputs are transformed from their exernal representations to an internal representation.
+Internal representations are staged for appraisal processing, such as populating an input queue.
+
+### **Phase 2**: Evidence Augmentation
+
+During Phase 2, Evidence inputs are added to a list that describes the Attester's actual state.
+These inputs are added with the Attester's authority.
+
+### **Phase 3**: Reference Values Corroboration and Augmentation
+
+During Phase 3, Reference Values inputs are compared with Evidence inputs.
+Reference Values inputs describe possible states of Attesters.
+If the actual state of the Attester is described by the possible Attester states, then the overlapping (corroborated) actual states are added to the Attester's actual state.
+These inputs are added with the Reference Value Provider's authority.
+
+### **Phase 4**: Endorsed Values Augmentation
+
+During Phase 4, Endorsed Values inputs containing conditions that describe expected Attester state are compared.
```

Comment:
Compared against whom ?

Reply:
Addressed in [4857331](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/232/commits/48573314c87a73fda94200a755c207e2efd21ec5)